src/averageAges.js

```diff
@@ -14,12 +14,31 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let menOnly = people.filter(({
+    sex,
+  }) => sex === 'm');
+
+  if (century) {
+    menOnly = menOnly.filter(({
+      died,
+    }) => (Math.ceil(died / 100) === century));
+  }
+
+  const menAgesSum = menOnly.reduce(
+    (
+      sum,
+      man,
+    ) => sum + (man.died - man.born), 0);
+  const menAverageAge = menAgesSum / menOnly.length;
+
+  return menAverageAge;
```

Comment:
no need for the separate variable if you return it from the function on the next line

```suggestion

  return menAgesSum / menOnly.length;
```